Kickstarter-iOS/Views/Controllers/PaymentMethodsViewController.swift

```diff
@@ -0,0 +1,107 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+internal final class PaymentMethodsViewController: UIViewController {
+
+  private let dataSource = PaymentMethodsDataSource()
+  private let viewModel: PaymentMethodsViewModelType = PaymentMethodsViewModel()
+
+  @IBOutlet private weak var headerLabel: UILabel!
+  @IBOutlet private weak var tableView: UITableView! {
+    didSet {
```

Comment:
I kinda like this way of in-lining the configuration of things. Just wondering if we've started doing this anywhere else? It just kind've break consistency.

Reply:
Yeah, I like this too way of configuring views too. But will change it to keep consistency.